### PR TITLE
[Refactor] Internet for 3d hand pose estimation

### PR DESCRIPTION
**Pull request by LareinaM**
## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmpose/evaluation/metrics/hand_metric.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+from ..functional import keypoint_epe
+
+
+@METRICS.register_module()
+class HandMetric(BaseMetric):
+
+    METRICS = {'MPJPE', 'MRRPE', 'Handedness_Acc'}
+
+    def __init__(self,
+                 modes: List[str] = ['MPJPE', 'MRRPE', 'Handedness_Acc'],
```
Comment (Tau-J) [inline mmpose/evaluation/metrics/hand_metric.py:19]:
```suggestion
                 modes: List[str] = ['MPJPE', 'MRRPE', 'HandednessAcc'],
```

File: mmpose/evaluation/metrics/hand_metric.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                keypoints_cam[..., 21:, :] -= keypoints_cam[..., 41, :]
+
+                pred_coords_cam[..., :21, 2] += abs_depth[0]
+                pred_coords_cam[..., 21:, 2] += abs_depth[1]
+                pred_coords_cam = pixel_to_camera(pred_coords_cam, focal[0],
+                                                  focal[1], principal_pt[0],
+                                                  principal_pt[1])
+
+                pred_coords_cam[..., :21, :] -= pred_coords_cam[..., 20, :]
+                pred_coords_cam[..., 21:, :] -= pred_coords_cam[..., 41, :]
```
Comment (Tau-J) [inline mmpose/evaluation/metrics/hand_metric.py:82]: Please avoid assuming the number of keypoints is 21

Comment (LareinaM) [reply mmpose/evaluation/metrics/hand_metric.py:82]: renamed metric to `InterHandMetric`

**Review by Tau-J — APPROVED** [review]
LGTM

**Pull request merged**